main.rb

```diff
@@ -0,0 +1,140 @@
+require './book'
+require './person'
+require './rental'
+require './student'
+require './teacher'
+def add_student
+  puts 'Name'
+  name = gets.chomp
+  puts 'Age'
+  age = gets.chomp
+  puts 'Has parent permission (Y/N)'
+  has_permission = gets.chomp
+  has_permission = has_permission == 'y'
+  student = Student.new(age, '', name, has_permission)
+  puts 'Student created successfully'
+  student
+end
```

Comment:
- [ ] This function receives more than a required number of arguments, it receives 4 but requires 2-3. I could not create a student object because I received an error for wrong number of arguments,